source/Cosmos.System2/Network/IPv4/OutgoingBuffer.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System.Collections.Generic;
+﻿using sys = System;
```

Comment:
Not sure about if this addition can break anything.
would love to hear your thoughts.